src/vivarium/framework/time.py

```diff
@@ -84,21 +89,21 @@ def simulant_next_event_times(self, index: pd.Index) -> pd.Series:
         """The next time each simulant will be updated."""
         if not self.population_view:
             raise ValueError("No population view defined")
-        return self.population_view.subview(["next_event_time"]).get(index)
+        return self.population_view.subview(["next_event_time"]).get(index).squeeze()
```

Comment:
Shouldn't this be `.squeeze{axis=1)` here to ensure we don't squeeze to a scalar if there is only one row?

Reply:
done